### PR TITLE
AP_HAL_ChibiOS: fix Matek bootloader to turn on switched supply during bootload to match default

### PR DESCRIPTION
**Pull request by Hwurzburg**
Makes video power pin on during bootloading to match default behavior after boot....this also makes these behave the same as with Betaflight software....tested on MatekH743-Wing

also adds a missing bootloader file

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/MatekH743/hwdef-bl.dat [hunk 1]
```diff
@@ -38,6 +38,9 @@ PA12 OTG_FS_DP OTG1
 PA13 JTMS-SWDIO SWD
 PA14 JTCK-SWCLK SWD
 
+# make sure Vsx is on during bootloader
+PD10  PINIO1 OUTPUT LOW
```
Comment (andyp1per) [inline libraries/AP_HAL_ChibiOS/hwdef/MatekH743/hwdef-bl.dat:42]: You should do the camera as well I think?

Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/MatekH743/hwdef-bl.dat:42]: Vsw by the way, and would be better if it was labelled something like VTX_SWITCH or something more descriptive

**Review by andyp1per — CHANGES_REQUESTED** [review]
I think probably we should do the camera pin as well as the Vsw pin, both are power as far as I can tell

**Comment by Hwurzburg** [comment]
It's not a power switch,it's a video mux and has no system impact that I can see,like power sequencing can have on getting the vtx ready to accept commands...but it can't hurt

File: libraries/AP_HAL_ChibiOS/hwdef/MatekH743/hwdef-bl.dat (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/MatekH743/hwdef-bl.dat:42]: done

**Comment by andyp1per** [comment]
> It's not a power switch,it's a video mux and has no system impact that I can see,like power sequencing can have on getting the vtx ready to accept commands...it does not matter on the boards where it controls that and does not help anything...HOWEVER, on the TE boards, some of them only pin out that signal, so it would be worth doing it....will change the TE one

Doesn't it mean that camera 2 is switched at boot rather than camera 1 if you don't do this? Anyway given that Matek document the default it would make sense to copy

**Comment by Hwurzburg** [comment]
Cameras are non-functional during the bootloader they don't work until the OSD chip has been initialized in the main code

**Review by andyp1per — APPROVED** [review]

**Review by MATEKSYS — APPROVED** [review]

**Review by tridge — APPROVED** [review]

**Pull request merged**